ner_v2/tests/test_number_detection.py

```diff
@@ -0,0 +1,223 @@
+from __future__ import absolute_import
+
+from django.test import TestCase
+import collections
+import datetime
+import pytz
+
+from ner_v2.detectors.numeral.utils import get_number_from_number_word
+from ner_v2.detectors.numeral.number.number_detection import NumberDetector
+
+NumberVariant = collections.namedtuple('NumberVariant', ['scale', 'increment'])
+
+
+class NumberDetectionTest(TestCase):
```

Comment:
Good work! 👍 
We should also try adding negative tests, edge cases and more examples per test

Reply:
Thanks